servicemodels/e2sm_rc/pdubuilder/E2SM-RcRanfunctionDefinition.go

```diff
@@ -0,0 +1,93 @@
+//  SPDX-FileCopyrightText: 2020-present Open Networking Foundation <info@opennetworking.org>
+//
+//  SPDX-License-Identifier: Apache-2.0
+
+package pdubuilder
+
+import (
+	e2smcommonies "github.com/onosproject/onos-e2-sm/servicemodels/e2sm_rc/v1/e2sm-common-ies"
+	e2smrcv1 "github.com/onosproject/onos-e2-sm/servicemodels/e2sm_rc/v1/e2sm-rc-ies"
+)
+
+func CreateE2SmRcRanfunctionDefinition(rfsn string, oID string, dscr string) (*e2smrcv1.E2SmRcRanfunctionDefinition, error) {
+
+	msg := &e2smrcv1.E2SmRcRanfunctionDefinition{
+		RanFunctionName: &e2smcommonies.RanfunctionName{
+			RanFunctionShortName:   rfsn,
+			RanFunctionE2SmOid:     oID,
+			RanFunctionDescription: dscr,
+		},
+	}
+
+	return msg, nil
+}
+
+func CreateE2SmRcRanfunctionDefinitionEventTrigger(rfsn string, oID string, dscr string, et *e2smrcv1.RanfunctionDefinitionEventTrigger) (*e2smrcv1.E2SmRcRanfunctionDefinition, error) {
```

Comment:
I would suggest to use complete variable name. oID should OID but rfsn should ranFunctionShortName. It is not clear what this abbreviation stands for.